test/scripts/test_bytecodecompare_prepare_report.py

```diff
@@ -14,7 +14,7 @@
 # pragma pylint: enable=import-error
 
 
-SMT_SMOKE_TEST_SOL_PATH = LIBSOLIDITY_TEST_DIR / 'smtCheckerTests/simple/smoke_test.sol'
+SMT_SMOKE_TEST_SOL_PATH = FIXTURE_DIR / 'smt_smoke_test.sol'
 SMT_SMOKE_TEST_SOL_CODE = load_libsolidity_test_case(SMT_SMOKE_TEST_SOL_PATH)
```

Comment:
```suggestion
SMT_SMOKE_TEST_SOL_CODE = load_fixture(SMT_SMOKE_TEST_SOL_PATH)
```

BTW, `load_libsolidity_test_case()`/`load_fixture()` are just convenience helpers that prepend `FIXTURE_DIR` or `LIBSOLIDITY_TEST_DIR` but I see that they're getting paths with them already prepended. Interestingly, it does not break only because the paths are absolute (so `prefix / path` equals `path`). But I should clean this up. I'll do this in a separate PR.

Reply:
Should I still apply this suggestion?

Reply:
Yeah. It won't hurt and it will at least be consistent with other calls in this module.